src/components/Accordion/story.js

```diff
@@ -0,0 +1,123 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import ExpansionPanel from '../ExpansionPanel';
+import Icon from '../Icon';
+import Accordion from './';
+import Readme from './README.md';
+
+storiesOf('Galaxies/Accordion', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
```

Comment:
The "Overview" story should be fully controlled with knobs (https://www.npmjs.com/package/@storybook/addon-knobs) for every prop.

The "Example" is where you can provide state and make interactive demos of various pieces of functionality.

Reply:
@pixelbandito The only prop I can think about is `style`.
`defaultExpandedPanel` is only used to set the initial state of the component. I could watch for its changes using `useEffect` and update the state. What do you think?

Reply:
Might be a crazy thought, but you could add a `key` based on the `defaultExpandedPanel` prop, and then React should remount the component of the user changes that one prop.